src/services/auth.js

```diff
@@ -48,6 +49,20 @@ passport.use(new HTTPSignatureStrategy(
       })
   }))
 
+passport.use(new ClientCertStrategy((fingerprint, info, done) => {
+  if (!config.auth.client_certificates_enabled) {
+    return done(new UnauthorizedError('Unsupported authentication method'))
+  }
+
+  Account.findByFingerprint(fingerprint)
+    .then(function (userObj) {
+      if (!userObj || userObj.is_disabled || userObj.fingerprint !== fingerprint) {
```

Comment:
You suggested I add the `!userObj.fingerprint` condition, but this sounds like `!fingerprint`. That would only happen if this implementation were to change without anyone noticing https://github.com/interledger/five-bells-shared/pull/50/files#diff-4d47c0b51e2406b055b5d1c2885e7f9fR96. I will add both cases though, as it just makes the check safer.